src/intro/architecture.md

```diff
@@ -0,0 +1,44 @@
+# Architecture Overview
+
+## 10,000 ft. view
+
+We want to simplify the lives of DevOps teams by providing an on-prem solution that can give full control over telemetry pipelines, while maintaining existing use of Observability Platforms and/or tools.
+
+![![System & Users](../media/arch-c1.png)](../media/arch-c1.png)
+
+## 1,000 ft. view
+
+Here's a brief diagram for how we are using our preferred technologies to build our infrastructure to help the DevOps community!
+
+[![Container Diagram](../media/arch-c2.png)](../media/arch-c2.png)
```

Comment:
I think a paragraph describing what the API is doing and what the Console does below here would be useful. 

I'd also say the MyDecisiveAI octopus outside of the k8s cluster with an arrow saying "monitors" and "configures" is confusing. It could give the impression that MyDecisive is a cloud services running elsewhere monitoring your deployment.